src/routes/Home/index.tsx

```diff
@@ -35,6 +36,7 @@ function Home(): ReactElement {
 
         <Card>
           <h2>Transactions to Sign</h2>
+          <PendingTxsList />
         </Card>
```

Comment:
It probably makes sense to create some kind of wrapper that we can pass the dimensions, title and module to here. Each component will have duplicate cards and titles otherwise.